charts/rancher-turtles/100.1.0+up0.5.0/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
This chart.lock file is weird, I have not seen this before. 
How was this generated? 
What is it used for?